lib/internal/http2/core.js

```diff
@@ -1506,6 +1506,13 @@ class Http2Stream extends Duplex {
       this.once('ready', this._destroy.bind(this, err, callback));
       return;
     }
+
+    const server = session[kServer];
+
+    if (err && server) {
+      server.emit('clientError', err, this);
```

Comment:
there is a `nextTick` on top.

`_destroy` is called synchronously. But the error of the `_destroy` callback will be emitted asynchronously, see https://github.com/nodejs/node/blob/master/lib/internal/streams/destroy.js#L32-L37.